plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -408,6 +430,16 @@ fu_dell_dock_ec_get_dock_info (FuDevice *device,
 	}
 	fu_device_set_version_lowest (device, self->ec_minimum_version);
 
+
+	/* TODO: Drop if minimum EC is set to 23+
+	 * Determine if the passive flow should be used when flashing
+	 */
+	if (fu_device_has_custom_flag (device, "try-passive-flow") &&
+	    fu_common_vercmp (self->ec_version, "00.00.00.23") >= 0) {
+		g_debug ("using passive flow");
+		self->passive_flow = PASSIVE_REBOOT_MASK;
+		fu_device_set_custom_flags (device, "skip-restart");
```

Comment:
could we just remove the NEEDS_REBOOT flag?

Reply:
Currently don't use the `NEEDS_REBOOT` device flag with this device.  I think the meaning of it might be a little diluted if it was used.  We're talking about a "device" reboot but the general use so far of that flag has been around a system reboot.